RecoMET/METAlgorithms/src/HcalHaloAlgo.cc

```diff
@@ -119,6 +127,66 @@ HcalHaloData HcalHaloAlgo::Calculate(const CaloGeometry& TheCaloGeometry, edm::H
 	  TheHcalHaloData.GetPhiWedges().push_back( wedge );
 	}
     }
+
+  // Don't use HF.
+  int maxAbsIEta = 29;
+  std::vector<const CaloTower*> SortedCaloTowers;
```

Comment:
please keep local variables names starting with lower case (CMS style); first letter capital is reserved for type names